lib/publiccloud/provider.pm

```diff
@@ -313,6 +313,9 @@ sub create_instances {
             # Install server's ssh publicckeys to prevent authenticity interactions
             assert_script_run(sprintf('ssh-keyscan %s >> ~/.ssh/known_hosts', $instance->public_ip));
         }
+        # Performance data: boottime
```

Comment:
$instance->wait_for_guestregister();

Reply:
`create_instance()` call's input parameter updated in those modules where it does _not_ followed by _wait_for_guestregister_ call.

Reply:
see i.e. this parameter's use in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16577#discussion_r1140070396